### ISSUE TITLE
Dev blog

### ISSUE THREAD

**Issue by donbonifacio** [puppet-coliseum/mothership#5]
Should we start a dev blog? The RFCs already have good material for it. If so, were? Should we build a jekyll site, use tumblr, ...?

The dev blog might bring additional people to the game.

**Comment by lamuria** [comment]
Yeah, we should start a dev blog. I prefer Jekyll.. but we can use tumblr or medium too.. It's ok for me any of these tools

**Comment by jqmtor** [comment]
People use Ghost a lot these days. I think it would be good to look into this one too. The main disadvantage I see is that we would need to host it somewhere, since we're not able to take advantage of GitHub for that purpose. On the hand, it has a built-in editor, making it a lot easier to write, people say. I don't know much about it but wanted to leave it here for reference.

https://ghost.org/

**Comment by donbonifacio** [comment]
Tried ghost, very simple and has medium-style good looks. Didn't like two things:
- Hosted version is very expensive (10$/month)
- Hosting ourselves wihouth github will cost money and we'll have additional things to worry about (like backups... I lost 2 great OB blogs...)

**Comment by jqmtor** [comment]
Jekyll it is! :+1:
